reconcile/utils/three_way_diff_strategy.py

```diff
@@ -61,7 +61,7 @@ def normalize_object(item: OR) -> OR:
         validate_k8s_object=False,
     )
 
-    annotations = n.body.get("metadata").get("annotations") or {}
+    annotations = n.body.get("metadata", {}).get("annotations", {})
```

Comment:
any edge case that `annotations` key exist but value is `None`?

Reply:
TBH, I don't know. I switched it back in 830733e